ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/DisplayGrid/DisplayGrid.qml

```diff
@@ -206,7 +206,7 @@ DisplayGridSample {
             ComboBox {
                 id: colorCombo2
                 property int modelWidth: 0
-                Layout.minimumWidth: modelWidth + leftPadding + rightPadding + indicator.width
+                Layout.minimumWidth: modelWidth + leftPadding + rightPadding
```

Comment:
```suggestion
                Layout.minimumWidth: modelWidth + leftPadding + rightPadding + (indicator ? indicator.width : 0)
```

I am not sure about the warnings, so this may not make sense.

Reply:
I feel like this is going to overcomplicate the code without contributing given that indicator is always null anyway

Reply:
That makes sense. Now that you've demo'd it and we can see it is always null, removing is right.